mathics/builtin/exptrig.py

```diff
@@ -4,7 +4,7 @@
 """
 Exponential, Trigonometric and Hyperbolic Functions
 
-Mathics basically supports all important trigonometric and hyperbolic functions.
+\Mathics basically supports all important trigonometric and hyperbolic functions.
```

Comment:
I believe will run into an escape deprecation message unless you preface the `"""` with and `r`, (for raw string) e.g. `
```
r"""

\Mathics ....
```